server/src/test/java/org/elasticsearch/common/logging/LoggersTests.java

```diff
@@ -58,36 +58,37 @@ public void testCheckRestrictedLoggers() {
 
     public void testSetLevelWithRestrictions() {
```

Comment:
why do we have the same method in both restricted-loggers and in server?

Reply:
there's a subtile difference between the two, the one in server tests an internal interface where restricted loggers are provided by the test. this is because we have the trace network flag `insecure_network_trace_enabled` enabled for server tests, which then disables restricted loggers.

this test is mostly there to have a test close to the sources and get immediate feedback.

the other one runs the same test, but with the restricted loggers defined for production code.